R/zchunk_L141.hfc_R_S_T_Y.R

```diff
@@ -51,67 +48,210 @@ module_emissions_L141.hfc_R_S_T_Y_DISABLED <- function(command, ...) {
-    A41.GWP <- get_data(all_data, "emissions/A41.GWP")
-    EDGAR_HFC125 <- get_data(all_data, "emissions/EDGAR_HFC125")
-    EDGAR_HFC134a <- get_data(all_data, "emissions/EDGAR_HFC134a")
-    EDGAR_HFC143a <- get_data(all_data, "emissions/EDGAR_HFC143a")
-    EDGAR_HFC152a <- get_data(all_data, "emissions/EDGAR_HFC152a")
-    EDGAR_HFC227ea <- get_data(all_data, "emissions/EDGAR_HFC227ea")
-    EDGAR_HFC23 <- get_data(all_data, "emissions/EDGAR_HFC23")
-    EDGAR_HFC236fa <- get_data(all_data, "emissions/EDGAR_HFC236fa")
-    EDGAR_HFC245fa <- get_data(all_data, "emissions/EDGAR_HFC245fa")
-    EDGAR_HFC32 <- get_data(all_data, "emissions/EDGAR_HFC32")
-    EDGAR_HFC365mfc <- get_data(all_data, "emissions/EDGAR_HFC365mfc")
-    EDGAR_HFC43 <- get_data(all_data, "emissions/EDGAR_HFC43")
+    EDGAR_sector <- get_data(all_data, "emissions/EDGAR/EDGAR_sector_fgas")
+    EDGAR_HFC125 <- get_data(all_data, "emissions/EDGAR/EDGAR_HFC125")
+    EDGAR_HFC134a <- get_data(all_data, "emissions/EDGAR/EDGAR_HFC134a")
+    EDGAR_HFC143a <- get_data(all_data, "emissions/EDGAR/EDGAR_HFC143a")
+    EDGAR_HFC152a <- get_data(all_data, "emissions/EDGAR/EDGAR_HFC152a")
+    EDGAR_HFC227ea <- get_data(all_data, "emissions/EDGAR/EDGAR_HFC227ea")
+    EDGAR_HFC23 <- get_data(all_data, "emissions/EDGAR/EDGAR_HFC23")
+    EDGAR_HFC236fa <- get_data(all_data, "emissions/EDGAR/EDGAR_HFC236fa")
+    EDGAR_HFC245fa <- get_data(all_data, "emissions/EDGAR/EDGAR_HFC245fa")
+    EDGAR_HFC32 <- get_data(all_data, "emissions/EDGAR/EDGAR_HFC32")
+    EDGAR_HFC365mfc <- get_data(all_data, "emissions/EDGAR/EDGAR_HFC365mfc")
+    EDGAR_HFC43 <- get_data(all_data, "emissions/EDGAR/EDGAR_HFC43")
     HFC_Inventory_GV <- get_data(all_data, "emissions/HFC_Inventory_GV")
 
-    # ===================================================
-    # TRANSLATED PROCESSING CODE GOES HERE...
-    #
-    # If you find a mistake/thing to update in the old code and
-    # fixing it will change the output data, causing the tests to fail,
-    # (i) open an issue on GitHub, (ii) consult with colleagues, and
-    # then (iii) code a fix:
-    #
-    # if(OLD_DATA_SYSTEM_BEHAVIOR) {
-    #   ... code that replicates old, incorrect behavior
-    # } else {
-    #   ... new code with a fix
-    # }
-    #
-    #
-    # NOTE: there are 'match' calls in this code. You probably want to use left_join_error_no_match
-    # For more information, see https://github.com/JGCRI/gcamdata/wiki/Name-That-Function
-    # NOTE: This code uses vecpaste
-    # This function can be removed; see https://github.com/JGCRI/gcamdata/wiki/Name-That-Function
-    # NOTE: This code uses repeat_and_add_vector
-    # This function can be removed; see https://github.com/JGCRI/gcamdata/wiki/Name-That-Function
-    # ===================================================
+    # Silence Package check notes
+    "." <- "year" <- "value" <- "GCAM_region_ID" <- "sector" <- "fuel" <- "service" <-
```

Comment:
In the future, these shouldn't be in quotes, and need to assign them all to `NULL`. I'll change.